nvpy/view.py

```diff
@@ -1832,15 +1832,16 @@ def activate_links(self):
         # List of Regex patterns to match various link types to be activated
         re_list = [
             r"(\[\[[^][]*\]\])",  # Inter-note Links
-            r"((?:https?|ftp|file)://[-A-Za-z0-9+&@#/%?=~_|!:,.;\(\)]*[A-Za-z0-9+&@#/%=~_|])",  # Http(s) / FTP / File Links
+#            r"((?:https?|ftp|file)://[-A-Za-z0-9+&@#/%?=~_|!:,.;\(\)]*[A-Za-z0-9+&@#/%=~_|])",  # Http(s) / FTP / File Links
+            r"\b((https?|ftp|file)://[-\w+&@#/%?=~|!:,.;]*[\w+&@#/%=~|])|(\[\[[^][]*\]\])",  # Http(s) / FTP / File Links
             r"(mailto:[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)",  # Mailto Links
             r"((?:tel|mid):[^\s]+)",  # Tel / Mid Links
             r"((?:thunderlink|irc|ircs|irc6)://[^\s]+)",  # Thunderbird and IRC Links
             r"(message:(?://)?(?:%3c|<).*(?:%3e|>))"  # Leopard Mail Message Links
         ]
 
         # Compile Regex into single pattern
-        pat = re.compile('|'.join(re_list))
+        pat = re.compile('|'.join(re_list), re.U)
```

Comment:
`re.U` is not necessary. I revert it before merge this MR.

> Note that for backward compatibility, the re.U flag still exists (as well as its synonym re.UNICODE and its embedded counterpart (?u)), but these are redundant in Python 3 since matches are Unicode by default for strings (and Unicode matching isn’t allowed for bytes).
> https://docs.python.org/3/library/re.html